src/odemis/gui/main_xrc.py

```diff
@@ -7509,6 +7509,79 @@ def __init_resources():
   </object>
 </resource>'''
 
+    ______img_icon_ico_chevron_left_png = b'''\
+\x89PNG\x0d
+\x1a
+\x00\x00\x00\x0dIHDR\x00\x00\x00
+\x00\x00\x00\x08\x08\x06\x00\x00\x00\xc0\xfan\xb6\x00\x00\x00\x88zTXtR\
+aw profile type exif\x00\x00x\xda]\x8eI\x0e\x85@\x08D\xf7\x9c\xc2#05\xc3\
+q\x8c\xd1\xe4\xdf\xe0\x1f_\xda\xd68\xbcEQ\x10R\x00\xeb\xff\xb7\xc1\xd4!\
+T\xd0\xe6ai\x86\x85\xa6&\xcfe\x02\x07\x82H\x8c\xd4k\xe9\xe0\xacB\xe5\xb8\
+\xda\xb3\x07\xe1a,\xc3Q\xefEU|\xd1\xc4\xc26Wwk\xb6\xd8\xc2\x95\xce\xab\xc8\
+\xa1$\x08\xc7V\x0f\xeb\xaf\xe4\xe3\xf2|}\xf5\x99\xc7\xfb\x00\xec\xfb\x86\
+-\x0c\x9d]\x0eM\x00\x00
+\x00iTXtXML:com.adobe.xmp\x00\x00\x00\x00\x00<?xpacket begin="\xef\xbb\
+\xbf" id="W5M0MpCehiHzreSzNTczkc9d"?>
+<x:xmpmeta xmlns:x="adobe:ns:meta/" x:xmptk="XMP Core 4.4.0-Exiv2">
+ <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+  <rdf:Description rdf:about=""
+    xmlns:exif="http://ns.adobe.com/exif/1.0/"
+    xmlns:tiff="http://ns.adobe.com/tiff/1.0/"
+   exif:PixelXDimension="10"
+   exif:PixelYDimension="8"
+   tiff:ImageWidth="10"
+   tiff:ImageHeight="8"
+   tiff:Orientation="1"/>
+ </rdf:RDF>
+</x:xmpmeta>
```

Comment:
This is a sign that you've edited the PNG and didn't "groom" it (ie, optimized compression and removing metadata).
You can do that with `./util/groom-img.py src/odemis/gui/img/icon`